ecs_fargate/README.md

```diff
@@ -168,6 +168,9 @@ Metrics are collected with [DogStatsD][13] through UDP port 8125.
 
 To send custom metrics by listening to DogStatsD packets from other containers, set the environment variable `DD_DOGSTATSD_NON_LOCAL_TRAFFIC` to `true` within the Datadog Agent container.
 
+**Note**: There is no provision for a host tag with Fargate, so in order to use a unique identifier for the submission of custom metrics set `DD_DOGSTATSD_TAGS` to inject the task_arn tag to all the dogstatsd metrics or set the agent dogstatsd tagger in orchestrator mode to automatically inject the `taks_arn` tag. This requires using origin detection(via UDS)to set the cardinality in the agent via `DD_DOGSTATSD_TAG_CARDINALITY`, see [DogStatsD over Unix Domain Socket][41]. (please note that implementing this setting can contribute to increased custom metric cardinality) 
```

Comment:
```suggestion
**Note**: There is no notion of a host tag with Fargate, so to provide a unique identifier for submitting custom metrics, set `DD_DOGSTATSD_TAGS` to inject the `task_arn` tag to all the DogStatsD metrics or set the Agent DogStatsD tagger in orchestrator mode to automatically inject the `taks_arn` tag. This requires using origin detection (via UDS) to set the cardinality in the Agent via `DD_DOGSTATSD_TAG_CARDINALITY`. For more information, see [DogStatsD over Unix Domain Socket][41]. (Implementing this setting can contribute to increased custom metric cardinality.) 
```

Reply:
We more context here on why you might want to do this.

Reply:
We may want to refer back to our docs that talk about why `host` tags are important. Something like this:

> **Note**:DogStatsD requires a unique identifier, typically [reserved for the host tag](https://docs.datadoghq.com/developers/metrics/dogstatsd_metrics_submission/#metric-tagging). There is no notion of a host tag with Fargate, so to provide a unique identifier for submitting custom metrics, set `DD_DOGSTATSD_TAGS` to inject the `task_arn` tag to all the DogStatsD metrics or set the Agent DogStatsD tagger in orchestrator mode to automatically inject the `taks_arn` tag. This requires using origin detection (via UDS) to set the cardinality in the Agent via `DD_DOGSTATSD_TAG_CARDINALITY`. For more information, see [DogStatsD over Unix Domain Socket][41]. (Implementing this setting can contribute to increased custom metric cardinality.)